vm/systemSmartContracts/delegationManager_test.go

```diff
@@ -1192,7 +1193,7 @@ func TestDelegationManagerSystemSC_ClaimMultipleDelegation(t *testing.T) {
 			GetCalled: func(key []byte) (vm.SystemSmartContract, error) {
 				return &mock.SystemSCStub{
 					ExecuteCalled: func(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
-						_ = d.eei.Transfer(args.RecipientAddr, args.CallerAddr, big.NewInt(10), nil, 0)
+						_ = d.eei.Transfer(args.CallerAddr, args.RecipientAddr, big.NewInt(10), nil, 0)
```

Comment:
is this change ok? Why the test passed before?

Reply:
because the test did not verify that the user gets the number of transfers he should